test/source/tests/browser-unit-tests/unit-ContactStore.js

```diff
@@ -531,3 +531,31 @@ BROWSER_UNIT_TEST_NAME(`ContactStore doesn't replace revoked key with older vers
   }
   return 'pass';
 })();
+
+BROWSER_UNIT_TEST_NAME(`ContactStore searchPubkeys { hasPgp: true } returns all keys`);
+(async () => {
+  const db = await ContactStore.dbOpen();
+  const contactABBDEF = await ContactStore.obj({
+    email: 'abbdef@test.com', pubkey: testConstants.abbdefTestComPubkey
+  });
+  const contactABCDEF = await ContactStore.obj({
+    email: 'abcdef@test.com', pubkey: testConstants.abcdefTestComPubkey
+  });
+  const contactABCDDF = await ContactStore.obj({
+    email: 'abcddf@test.com', pubkey: testConstants.abcddfTestComPubkey
+  });
+  const contactABDDEF = await ContactStore.obj({
+    email: 'abddef@test.com', pubkey: testConstants.abddefTestComPubkey
+  });
+  await ContactStore.save(db, [contactABBDEF, contactABCDEF, contactABCDDF, contactABDDEF]);
+  const foundKeys = await ContactStore.searchPubkeys(db, { hasPgp: true });
```

Comment:
Actually, we can shortcut the `ContactStore.searchPubkeys(db, { hasPgp: true })` to simply load all keys from `pubkeys` store instead of using `search` index on emails. There are no other calls on `searchPubkeys` in the project as of now.

Reply:
Understood, ok to do